betydb2geojson.py

```diff
@@ -171,3 +172,4 @@ def convert() -> None:
 
 if __name__ == "__main__":
     convert()
+    sys.exit()
```

Comment:
I don't think this is required.

Reply:
Adding this to a new ticket that's also to investigate using `csvstack` per your suggestion

Reply:
Added issue https://github.com/AgPipeline/issues-and-projects/issues/257 for this. It was put in there when I was having problems with the return codes not being what was expected - I don't remember if this resolved the issue or not, added issue to backlog since it doesn't negatively impact anything to have it in place

Reply:
Yup. Makes sense.